go.sum

```diff
@@ -218,8 +218,8 @@ github.com/googleapis/gax-go/v2 v2.0.5/go.mod h1:DWXyrwAJ9X0FpwwEdw+IPEYBICEFu5m
 github.com/grpc-ecosystem/grpc-gateway v1.14.6/go.mod h1:zdiPV4Yse/1gnckTHtghG4GkDEdKCRJduHpTxT3/jcw=
 github.com/grpc-ecosystem/grpc-gateway v1.16.0 h1:gmcG1KaJ57LophUzW0Hy8NmPhnMZb4M0+kPpLofRdBo=
 github.com/grpc-ecosystem/grpc-gateway v1.16.0/go.mod h1:BDjrQk3hbvj6Nolgz8mAMFbcEtjT1g+wF4CSlocrBnw=
-github.com/hacbs-contract/enterprise-contract-controller/api v0.0.0-20221123200519-210c9331b2e2 h1:dIjGMlLHfFsGk6FqLpv7foxGU113bjZW3cXrurFMsjw=
-github.com/hacbs-contract/enterprise-contract-controller/api v0.0.0-20221123200519-210c9331b2e2/go.mod h1:KIVowf1Q+rkLyFdcquUHT/4QIov9j5C4TgGJ/jgT3io=
+github.com/hacbs-contract/enterprise-contract-controller/api v0.0.0-20221215171221-93099d5a1eff h1:UnXu7RA6twiM8nXbx/LZH/vc41RKNUvnPRFCmCqIv7Q=
+github.com/hacbs-contract/enterprise-contract-controller/api v0.0.0-20221215171221-93099d5a1eff/go.mod h1:KIVowf1Q+rkLyFdcquUHT/4QIov9j5C4TgGJ/jgT3io=
```

Comment:
I didn't check the digests, but beware there was a fix merged just a few seconds ago. Maybe it needs to refresh.

Reply:
The fix included updated CRDs, see https://github.com/hacbs-contract/enterprise-contract-controller/pull/48